scripts/get_max_locators.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+readonly THIS_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd | xargs realpath)"
+readonly ROOT_DIR="${THIS_DIR}/.."
+readonly INTERNAL_DIR="${ROOT_DIR}/internal"
+
+function process_directory() {
+    local path="$1"
+
+    local dir=$(basename "${path}")
```

Comment:
Call this `dirname`.
Like "filename" we usually use "dirname" when the value is a name (instead of a path) and then "dir" when it is a path.
This is a pretty loose convention, but nice when done consistently.